.github/workflows/build.yml

```diff
@@ -55,7 +55,17 @@ jobs:
           run: dotnet build SentryNoSamples.slnf -c Release /p:CopyLocalLockFileAssemblies=true
 
         - name: Test
-          run: dotnet test SentryNoSamples.slnf -c Release --no-build -l "GitHubActions;report-warnings=false"
+          run: dotnet test SentryNoSamples.slnf -c Release --no-build -l "GitHubActions;report-warnings=false" -l "trx;LogFileName=${{ matrix.os }}-test-results.trx"
+
+        - name: Create Test Report
+          uses: phoenix-actions/test-reporting@4b942144e31c4039ac808d9b29945864b628a574
+          if: success() || failure()
```

Comment:
do we need this condition?

Reply:
i added a comment `# run this step even if previous step failed`